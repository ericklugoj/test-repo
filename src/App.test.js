import { render, screen } from '@testing-library/react';
import App from './App';

test('renders learn react link', () => {
  render(<App />);
  const linkElement = screen.getByText(/test repo/i);
  expect(linkElement).toBeInTheDocument();
});

test('match snapshot', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
