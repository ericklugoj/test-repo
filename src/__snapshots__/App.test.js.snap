// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<div>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <img
        alt="logo"
        class="App-logo"
        src="logo.svg"
      />
      <p>
        Edit 
        <code>
          src/App.js
        </code>
         and save to reload.
      </p>
      <a
        class="App-link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        TEST REPO
      </a>
    </header>
  </div>
</div>
`;
